autoload/db_ui/schemas.vim

```diff
@@ -85,11 +85,52 @@ let s:mysql = {
       \ 'layout_flag': '\\G',
       \ 'quote': 0,
       \ }
+
+let s:oracle_args = 'echo "'.join(
+      \    [
+           \  'SET linesize 4000',
+           \  'SET pagesize 4000',
+           \  'COLUMN owner FORMAT a20',
+           \  'COLUMN table_name FORMAT a25',
+           \  'COLUMN column_name FORMAT a25',
+           \  '%s',
+      \    ],
+      \    ";\n"
```

Comment:
All queries require a semicolon and new line between entries. This essentially simulates piping in an entire script file from the commandline.

Adding or removing statements is a matter of adding them and removing them from the list. `'%s` is the query we actually want to ask the server, the rest is just setup.

Reply:
@kristijanhusak should I keep this `join`? I removed the others with the latest commit, but wasn't sure about this one since it handled some actual command formatting

Reply:
You can leave it.